Support_Your_Locals/Models/ViewModels/UserRegisterModel.cs

```diff
@@ -13,6 +13,9 @@ public class UserRegisterModel
         public DateTime BirthDate { get; set; }
         [Required(ErrorMessage = "Please enter your email address")]
         public string Email { get; set; }
+        [Required(ErrorMessage = "Please enter your password")]
+        public string Passhash { get; set; }
```

Comment:
If I understood correctly, at this stage password is not yet hashed, maybe it would be better to just name it 'Password'?